wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -28,6 +28,8 @@ These terms are frequently appearing when getting in touch with osu!catch diffic
 -   **Hyperjump:** A more exaggerated spacing which cannot be caught by normal dashing. During play, hyperdash will be triggered between the two objects, characterised by a coloured outline on the first object.
 -   **Edge Dash:** A very large spacing between two objects where the required Trigger Distance is not reached, and as such, a hyperjump is not generated. The first object must be caught with the edge of the plate in order to catch the second object at all.
 -   **Trigger distance:** The minimum spacing between two objects at which a hyperdash is generated between them.
+-   **Equivalent BPM:** A BPM value in which a certain snap is time-equivalent (same duration in ms) to a paired reference BPM/snap. This is calculated as Equivalent BPM = (`Reference BPM`)*(`Desired Snap`)/(`Reference Snap`).
```

Comment:
I am yet to figure out how this formula is even supposed to be used. Remove the entire thing and rather work with clean-cut millisecond threshold. This saves people a lot of headache.